README.md

```diff
@@ -104,50 +93,43 @@ To enable the Heat service, we simply need to set the Heat service to enabled in
 The following snippet is taken from the `OpenStackControlPlane` Custom Resource:
 
 ```yaml
-heat:
-  enabled: true # <<-- Enable the Heat service by setting `enabled: true`
-  template:
-    customServiceConfig: "# add your customization here"
-    databaseInstance: openstack
-    databaseUser: "heat"
-    rabbitMqClusterName: rabbitmq
+❯ oc get openstackcontrolplane openstack -o yaml | yq .spec.heat
+enabled: true #<-- This needs to be set to true for Heat to be deployed
+template:
+  customServiceConfig: '# add your customization here'
+  databaseInstance: openstack
+  databaseUser: heat
+  debug:
+    dbSync: false
+  heatAPI:
+    containerImage: quay.io/podified-antelope-centos9/openstack-heat-api:current-podified
+    customServiceConfig: '# add your customization here'
+    debug:
+      service: false
+    replicas: 1
+    resources: {}
+  heatCfnAPI:
+    containerImage: ""
```

Comment:
we can probably omit containerImage, customServiceConfig and debug because these are just defining the default values and redundant.